packages/kit/src/exports/node/polyfills.js

```diff
@@ -21,16 +26,21 @@ const globals = {
 };
 
 // exported for dev/preview and node environments
-// TODO: remove this once we only support Node 18.11+ (the version multipart/form-data was added)
 /**
  * Make various web APIs available as globals:
  * - `crypto`
- * - `fetch`
- * - `Headers`
- * - `Request`
- * - `Response`
+ * - `fetch` (only in node 18.11-)
+ * - `Headers` (only in node 18.11-)
+ * - `Request` (only in node 18.11-)
+ * - `Response` (only in node 18.11-)
```

Comment:
I would consider 18.11- to include 18.11. maybe we can write it differently to be more clear
```suggestion
 * - `fetch` (only in node < 18.11)
 * - `Headers` (only in node < 18.11)
 * - `Request` (only in node < 18.11)
 * - `Response` (only in node < 18.11)
```